admin_solutions/authentication.adoc

```diff
@@ -0,0 +1,166 @@
+= Authentication
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+You can configure many sources of authentication, as defined in
+link:../install_config/configuring_authentication.html[Configuring Authentication].
+For now, this topic focuses on GitHub as a source of authentication, but more
+providers will be added in a future release.
+
+[[github-auth]]
+== GitHub Authentication
+
+GitHub uses OAuth, and you can integrate your OpenShift cluster to use that
+OAuth authentication. OAuth basically facilitates a token exchange flow.
+
+Configuring GitHub authentication allows users to log in to OpenShift with their
+GitHub credentials. To prevent anyone with a GitHub user ID to log in to your
+OpenShift cluster, you can restrict users to require their membership in GitHub
+organizations.
```

Comment:
That last part might need to be a bit more precise:

"To prevent anyone with any GitHub user logging in to your OpenShift cluster, you can restrict access to only those  in certain GitHub organizations."